src/Symfony/Component/Mailer/Transport/Smtp/Stream/SocketStream.php

```diff
@@ -80,6 +80,19 @@ public function disableTls(): self
         return $this;
     }
 
+    public function disableTlsPeerVerification(): self
```

Comment:
I think we don't need this method: let's use get/setStreamOptions directly instead